lib/queue.js

```diff
@@ -276,6 +276,37 @@ class Queue extends Emitter {
     });
   }
 
+  _getJobsByIds(ids) {
+    // If ids is a Set (i.e. returned from Queue._scanForJobs), convert it to an
+    // Array.
+    const idsArray = Array.from(ids);
+    if (!idsArray.length) {
+      return Promise.resolve([]);
+    }
+
+    // We need to re-ensure the queue is commandable, as we might be shutting
+    // down during this operation.
+    return this._commandable().then((client) => {
+      const got = helpers.deferred();
+      const commandArgs = [this.toKey('jobs')].concat(idsArray, got.defer());
```

Comment:
[Related](https://github.com/bee-queue/arena/pull/38). I'm adding this functionality, but it won't be used in Arena for them time being.